quickstart/101-analysis-services-create/main.tf

```diff
@@ -0,0 +1,22 @@
+resource "random_pet" "rg_name" {
+  prefix = var.resource_group_name_prefix
+}
+
+resource "azurerm_resource_group" "rg" {
+  name     = random_pet.rg_name.id
+  location = var.resource_group_location
+}
+
+resource "azurerm_analysis_services_server" "server" {
+  name                = var.server_name
```

Comment:
Could we add a random string as server name's prefix here to avoid potential naming conflict?